chrome/locale/en-US/zotero/zotero.ftl

```diff
@@ -516,6 +516,9 @@ expand-all-sections =
 abstract-field =
     .placeholder = Add abstract…
 
+tag-field =
+    .placeholder = Edit tag...
```

Comment:
This can just be "Tag".

Reply:
Though it's kind of distracting visually even with just "Tag", since it appears for every single new row. Can we just use an ARIA label here?

![Screenshot 2024-07-16 at 1 42 04 AM](https://github.com/user-attachments/assets/c72afad0-9035-4f6e-b705-f5614bb4b2e5)

Reply:
For sure! There was a small part of me that always though that maybe some kind of placeholder would be good to not have an empty input field but `aria-label` (or maybe `aria-description` - let me try which is better) would also work

Reply:
I moved it to a general string, so we can always change this later.